CMakeLists.txt

```diff
@@ -312,9 +312,7 @@ endif()
 # Backend logic
 #--------------------------------------------------------------------
 ## default math backend is 2 for 32-bit and 64-bit integers and 4 for 128-bit integers
```

Comment:
please correct this comment if the default is always "MATHBACKEND = 4".

Reply:
Fixed